pkg/controllers/management/eksupstreamrefresh/eks_upstream_refresh.go

```diff
@@ -95,14 +98,26 @@ func (e *eksRefreshController) refreshClusterUpstreamSpec(cluster *mgmtv3.Cluste
 	}
 
 	upstreamSpec, err := GetComparableUpstreamSpec(e.secretsCache, cluster)
-	if err != nil {
+	if err != nil && upstreamSpec == nil {
+		var statusErr error
+		cluster = cluster.DeepCopy()
+		apimgmtv3.ClusterConditionUpdated.False(cluster)
+		apimgmtv3.ClusterConditionUpdated.Message(cluster, "[Syncing error: syncing broken] "+err.Error())
```

Comment:
Yes, I agree that we need a better phrase.

There are two reasons for a "Syncing error":
- When we sync, we noticed that one or more of the node groups are in a state that cannot be recovered from. The syncing of the EKS state still occurs here.
- When we sync, that actual syncing does not occur because of some type of problem.

Perhaps "syncing failed" would be better?